docs/ispc.rst

```diff
@@ -880,6 +880,11 @@ These flags enable developers to collect profile data from representative progra
 runs and use it to guide compiler optimizations, potentially improving
 performance.
 
+New Architecture Support:
```

Comment:
OK, I see it in the docs :) So we'll need to rebuild LLVM with RISC-V support moving forward and and add it to the release binaries. I'll take care of this.

Reply:
Yes. With the rebuilt LLVM, it will be possible to move the `risc-v.yml` job to the general ispc-ci test workflow.